freemius-stubs.php

```diff
@@ -5052,9 +5052,9 @@ function is_migration()
      * @param int         $blog_id
      *
      * @return array {
-     *      @var bool   $success
-     *      @var string $error
-     *      @var string $next_page
+     *      @type bool   $success
+     *      @type string $error
+     *      @type string $next_page
```

Comment:
WordPress core uses `@type` for array elements.

Reply:
done